include/deal.II/base/point.h

```diff
@@ -185,7 +185,7 @@ class Point : public Tensor<1, dim, Number>
    * @note This function can also be used in CUDA device code.
    */
   DEAL_II_CUDA_HOST_DEV Number
-                        operator()(const unsigned int index) const;
+  operator()(const unsigned int index) const;
```

Comment:
I think that the indentation script wants it just the way it was.

Reply:
This is really hard since my editor has been configured to indent on save following the .clang-format file in the project's root. I had to rename the .clang-format, work on my edit, but it wouldn't rebase until I had restored .clang-format.   

Why are the rules in the .clang-format file format provided in the project  not adhered to in the header files?

Reply:
I don't know what you mean. We apply clang-format everywhere. Which version of clang-format are you using? It needs to be 6.0.

Reply:
The clang-format version compatibility can be specified in the `.clang-format` file. My binary is v9.0

Reply:
I think if you wanted to propose a patch for that, that would be fine -- but I have no trust in that actually working. I would be very surprised if the 9.0 binary with the 6.0 setting produces the same indentation as 6.0.